.gitignore

```diff
@@ -11,6 +13,8 @@
 # Snapshots
 ## ... large
 *.full.snap
-
-#  ... unreviewed
+## ... unreviewed
```

Comment:
Why this comment exist and has been moved?

Reply:
the headers were not consistent. It is supposed to:
- Level 1: Snapshots
  - level 2: large
  - level 2: unreviewed
  
But instead, it was:
- Level 1: Snapshots
   - level 2: large
- Level 1: unreviewed